README.md

```diff
@@ -41,3 +40,11 @@ Octopus.Server.exe service --instance <instance> --start --nologo --console
 
 ** Ensure you update your build to the latest Calamari or revert to the bundled package when you upgrade Octopus Server **
 
+## Releasing
+
+After you finish merging to master to tag the Calamari NuGet package:
+
+* On your terminal, checkout `master` and `git pull` for good measure
+* Run `git tag` and scroll to the bottom of the list to get the last known tag
```

Comment:
Or just check the last build on `master` as it will be pre-release version of the next `major.minor.patch` version